samples/web/content/apprtc/js/appwindow.js

```diff
@@ -0,0 +1,75 @@
+/*
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+/* More information about these options at jshint.com/docs/options */
+// Variables defined in and used from main.js.
+/* globals randomString, initialize */
+/* exported params */
+'use strict';
+
+// Provide params var, provided inline in index.html.
+var params;
+
+// Generate random room id and connect.
+var room = randomString(9);
```

Comment:
Seems unnecessary to save it in a new var, e.g. you can do 
params.roomId = randomString(9)